js/editable-css.js

```diff
@@ -7,28 +7,29 @@
 
     var exampleChoiceList = document.getElementById('example-choice-list');
     var exampleChoices = exampleChoiceList.querySelectorAll('.example-choice');
-    var header = document.querySelector('header');
     var initialChoice = 0;
     var originalChoices = [];
-    var output = document.getElementById('output');
 
     /**
      * Enables and initializes the live code editor
      */
     function enableLiveEditor() {
+        var header = document.querySelector('header'),
+            output = document.getElementById('output');
+
         header.classList.remove('hidden');
         exampleChoiceList.classList.add('live');
         output.classList.remove('hidden');
 
-        for (var i = 0, l = exampleChoices.length; i < l; i++) {
-            var exampleChoice = exampleChoices[i];
+        for (var i = 0, choiceContainer, choiceCode; i < exampleChoices.length; i++) {
```

Comment:
Nit, code style: I would not suggest this code style, rather declare the variables inside the loop explicitly. Also, cache the length of the `Array` i.e.

```
for (var i = 0 l = exampleChoices.length, ; i < l; i++) {
    var choiceContainer = exampleChoices[i];
    var choiceCode = choiceContainer.querySelector('code');
```